swaymsg/main.c

```diff
@@ -326,6 +326,14 @@ static void pretty_print_tree(json_object *obj, int indent) {
 
 	printf("\n");
 
+    json_object *floating_nodes_obj;
+    json_bool floating_nodes = json_object_object_get_ex(obj, "floating_nodes", &floating_nodes_obj);
+    if (floating_nodes) {
+        size_t len = json_object_array_length(floating_nodes_obj);
+        for (size_t i = 0; i < len; i++) {
+            pretty_print_tree(json_object_array_get_idx(floating_nodes_obj, i), indent + 1);
```

Comment:
Can we print these after the regular `nodes`?

Reply:
Done